styles.css

```diff
@@ -1004,3 +1004,11 @@ section.intro-section .page-brk-dp {
 article.dark-bg:hover div.project {
   display: flex;
 }
+
+button:focus {
+  border: 3px solid #e24e5b;
+}
+
+footer img[alt] {
+  color: #fff;
+}
```

Comment:
- [ ] In the left bottom corner of your page(footer), the following icon doesn't match with the design. Please make the icon a little bit bigger to match the design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![f1](https://user-images.githubusercontent.com/70483835/155180615-70c9ca75-6079-444a-a539-3bcd5956de70.png) | ![f2](https://user-images.githubusercontent.com/70483835/155180641-e6be5588-aeff-4786-b8bf-8650ed7cf2f2.png)